README.md

```diff
@@ -1,10 +1,90 @@
-# Terraform Provider Project
+# Terraform Provider for Artifactory Project
 
 [![Actions Status](https://github.com/jfrog/terraform-provider-project/workflows/release/badge.svg)](https://github.com/jfrog/terraform-provider-project/actions)
 [![Go Report Card](https://goreportcard.com/badge/github.com/jfrog/terraform-provider-project)](https://goreportcard.com/report/github.com/jfrog/terraform-provider-project)
-[![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/jfrog/terraform)
 
-To use this provider in your Terraform module, follow the documentation [here](https://registry.terraform.io/providers/jfrog/project/latest/docs).
+## Quick Start
+
+Create a new Terrform file with `project` resource (also see [sample.tf](./sample.tf)):
+
+<details><summary>HCL Example</summary>
```

Comment:
This make it a collapsible section

Reply:
Can you add the reminder for contributors, to re-generate documentation with tfplugindocs, when they make changes?

Reply:
something like this https://github.com/jfrog/terraform-provider-xray/blob/f4ceb6f42d1a7cd8fd853dc2b0220f8a893a75a8/README.md?plain=1#L64